components/script/canvas_state.rs

```diff
@@ -292,6 +292,15 @@ impl CanvasState {
         *self.state.borrow_mut() = CanvasContextState::new();
     }
 
+    pub(crate) fn reset_bitmap(&self) {
+        if !self.is_paintable() {
+            return;
+        }
+
+        let rect = Rect::new(Point2D::zero(), self.size.get().to_f32());
+        self.send_canvas_2d_msg(Canvas2dMsg::ClearRect(rect));
```

Comment:
Nit: This can be a little more concise using `into()`

```suggestion
        self.send_canvas_2d_msg(Canvas2dMsg::ClearRect( self.size.get().to_f32().into()));
```

Reply:
Thanks. Missed this solution.